bin/cli.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const args = require('args');
+const setup = require('../src/setup');
+const mergeAccepted = require('../src/merge-accepted');
+const reviews = require('../src/reviews');
+
+args
+  .command('setup', 'Fork and clone all the fusionjs repos', setup)
+  .command('mergeAccepted', 'Merge all your accepted PRs', mergeAccepted)
+  .command('reviews', 'List all PRs needing review', reviews);
```

Comment:
Doesn't args require bin files for these?

Reply:
https://github.com/leo/args#commandname-description-init-aliases

Reply:
I went with the `init` function option rather than the bin option. I prefer being able to trace the code. Which reminds me, I need to move the promise rejection handling into this file